engines/python/setup/djl_python/tests/test_test_model.py

```diff
@@ -110,7 +109,6 @@ def test_all_code_chat(self):
     def test_with_env_chat(self):
         envs = {
             "OPTION_MODEL_ID": "TheBloke/Llama-2-7B-Chat-fp16",
```

Comment:
HF_MODEL_ID

Reply:
We should never use `OPTION_MODEL_ID`, I cannot think of a real use case that requires OPTION_MODEL_ID